src/main/java/software/amazon/lambda/snapstart/ByteCodeIntrospector.java

```diff
@@ -80,6 +83,16 @@ boolean implementsLambdaInterface(XClass xClass) {
         return false;
     }
 
+    boolean isLambdaHandlerParentClass(XClass xClass) {
+        lambdaHandlerParentsDatabase = Global.getAnalysisCache().getDatabase(LambdaHandlerParentsDatabase.class);
+        for (String parentClass : lambdaHandlerParentsDatabase.getParentClasses()) {
+            if (parentClass.equals(xClass.toString())) {
+                return true;
+            }
```

Comment:
Just a small comment here, but considering that `getParentClasses()` returns a `List`, I think we can directly do 
`return lambdaHandlerParentsDatabase.getParentClasses().contains(xClass.toString())`
instead of iterating ourselves.

Reply:
Makes sense. Thanks.